src/rgw/rgw_sal.h

```diff
@@ -336,6 +336,8 @@ class Store {
     const DoutPrefixProvider* dpp, rgw::sal::Object* obj, rgw::sal::Object* src_obj, 
     rgw::notify::EventType event_type, rgw::sal::Bucket* _bucket, std::string& _user_id, std::string& _user_tenant,
     std::string& _req_id, optional_yield y) = 0;
+    /** Get a @a NotificationConfig object for global topic configuration */
+    virtual std::unique_ptr<NotificationConfig> get_notification_config(const std::string& tenant, std::optional<const std::string> subscription) = 0;
```

Comment:
i think we should probably call that `get_topic_config()`. the global config objects are "topic config" and the per-bucket ones are "notification config'

probably good to remove the `subscription` parameter.
the subscription mechanism (a.k.a. pubsub) is going to be deprecated.

Reply:
I don't have a strong opinion on the name; I picked 'get_notification_config' because that's what it returned.  I do wonder if 'get_topic_config' is less discoverable?  Obviously the comment will help.

As for `subscription`, I can't remove it until `GetSubCR` is gone, or else it won't build.  I'm happy to remove it once that's gone.